support-workers/src/main/scala/com/gu/support/workers/lambdas/CreateZuoraSubscription.scala

```diff
@@ -43,19 +44,43 @@ class CreateZuoraSubscription(servicesProvider: ServiceProvider = ServiceProvide
   ): FutureHandlerResult = {
     val now = () => DateTime.now(DateTimeZone.UTC)
     val today = () => LocalDate.now(DateTimeZone.UTC)
+    val promotionService = services.promotionService
+    val redemptionService = services.redemptionService
+    val zuoraService = services.zuoraService
+    val isTestUser = state.user.isTestUser
+    val config: ZuoraConfig = zuoraConfigProvider.get(isTestUser)
+    doStepsFromDeps(state, requestInfo, now, today, promotionService, redemptionService, zuoraService, config)
+  }
+}
+object CreateZuoraSubscription {
+  import com.gu.FutureLogging._
+
+  def doStepsFromDeps(
```

Comment:
this needs a better name, there's already servicesHandler (which is an arbitrary name), I needed another arbitrary name that somehow says that this does the same things, but has more limited parameters specific to this lambda